dotcom-rendering/src/web/lib/braze/buildBrazeMessages.ts

```diff
@@ -1,7 +1,7 @@
 import { getCookie, log, storage } from '@guardian/libs';
+import type { BrazeMessagesInterface } from '@guardian/braze-components/logic';
```

Comment:
Is this import used?

Reply:
Yes, on line 46. It used to be bundled with the `class` imports, so we extracted it explicitely as a `type`.